directorywatcher/doc.go

```diff
@@ -0,0 +1,3 @@
+// Package directorywatcher provides a go implementation of baseplate's directorywatcher:
+// https://baseplate.readthedocs.io/en/stable/api/baseplate/lib/dir_watcher.html
```

Comment:
I get a 404 for this link. It could be useful to document the purpose of the package here separately.